### PR TITLE
Update action menu items in email plans page

### PR DESCRIPTION
**Pull request by daledupreez**
#### Changes proposed in this Pull Request

This PR does the following:
* Makes the Google Admin console link visible to admin users only
* Adds the Edit item for email forwards
* Updates the titles for the Titan links to have "View" as a prefix
* Updates the hover styles for the icons in the action menu to be light (note the Google and Titan icons are not plain white, as that would lose some texture for those icons)

**Note:** This PR depends on #52912 for some general style fixes.

#### Testing instructions

* Run this branch locally or open the [live branch](https://calypso.live/?branch=update/email-plans-action-menu-items)
* Navigate to the email plan page for a domain with a Google Workspace or G Suite subscription where one of the mailboxes is not an admin.
* Verify that the menu for that mailbox does _not_ include the "View Admin" menu item.
* Hover over the menu items and verify that the icons are light and recognisable when you hover.
* Repeat the exercise for a Google Workspace/G Suite mailbox that is an admin, but verify that the "View Admin" menu item does appear.
* Navigate to the email plan page for an email forwarding account.
* Verify that a menu is present for each mailbox, and includes the "Edit" menu item.
* Hover over the "Edit" menu item, and verify that the icon is white on hover.
* Click on the "Edit" menu item, and verify that you're taken to the email forwarding management page.
* Navigate to the email plan page for a domain with a Professional Email subscription.
* Open the menu for a mailbox.
* Verify that the menu items all start with "View ".
* Hover over the menu items and verify that the icons are recognisable and much lighter.

##### Removed "View Admin" item for Google
Screenshot 2021-05-18 at 09 29 56

##### New "Edit" menu item for email forwards (with hover)
Screenshot 2021-05-18 at 09 31 37

Screenshot 2021-05-18 at 09 31 52

##### New display for existing icons on hover
Screenshot 2021-05-18 at 09 27 43

Screenshot 2021-05-18 at 09 28 48

Related to #52912

### CODE REVIEW

**Comment by daledupreez** [comment]
@saygunnyc, I am pinging you here for a call on how best to handle the hover state for the non-flat icons we're using for the Google and Titan links. I've implemented the hover state using some filters to produce light grey colours with some of the icon textures retained.

File: client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx [hunk 1]
```diff
@@ -177,43 +180,47 @@ const getTitanMenuItems = ( email, translate ) => {
 			href: getTitanEmailUrl( email ),
 			image: titanMailIcon,
 			imageAltText: translate( 'Titan Mail icon' ),
-			title: translate( 'Mail', {
-				comment: 'This refers to the Email application (i.e. the webmail) of Titan',
+			title: translate( 'View Mail', {
+				comment: 'View the Email application (i.e. the webmail) for Titan',
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx:184]: I'm not sure if that's a valid assumption but I feel those changes will make those comments less clear to translators. `This refers to` or maybe `'Mail' refers to` seemed more appropriate to me but I could be totally wrong, so I'll let you decide 🙂.

File: client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx [hunk 2]
```diff
@@ -177,43 +180,47 @@ const getTitanMenuItems = ( email, translate ) => {
...
+				comment: 'View the Calendar application for Titan',
 			} ),
 		},
 		{
 			href: getTitanContactsUrl( email ),
 			image: titanContactsIcon,
 			imageAltText: translate( 'Titan Contacts icon' ),
-			title: translate( 'Contacts', {
-				comment: 'This refers to the Contacts application of Titan',
+			title: translate( 'View Contacts', {
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx:199]: I've already mentioned it internally but I wonder whether `Open [Contacts]` or Go to [Contacts] wouldn't be more appropriate or accurate.

**Review by stephanethomas — APPROVED** [review]
I've only found a small issue with email forwards but otherwise this works great:

> Click on the "Edit" menu item, and verify that you're taken to the email forwarding management page.

This opens a new tab instead of continuing in the same one. This link shouldn't be considered external. Feel free to merge that pull request once this has been fixed.


> Navigate to the email plan page for a domain with a Professional Email subscription.

I've gave a try to all three `View [...]` options, and I've noticed that the email address is only prefilled for `View Mail`. We should probably address that in the future.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5922816

Thank you @daledupreez for including a screenshot in the description! This is really helpful for our translators.

**Comment by saygunnyc** [comment]
> @saygunnyc, I am pinging you here for a call on how best to handle the hover state for the non-flat icons we're using for the Google and Titan links. I've implemented the hover state using some filters to produce light grey colours with some of the icon textures retained.

Hey @daledupreez, thanks for the ping!
Overall, we really should be following the same principles we use for all overflow menu items. However, as you mentioned, I noticed that the Google and Titan icons are not designed to be flat. 

That being said, I converted them all to be flat. Please see below the images for reference to the states. Also attaching both default and hover states of SVG icons as a .zip file. Can we try these? It should resolve the issue. Let me know if not...

OverflowMenu_Hover

Google_Full Icons

Titan_Full_Icons

[Google_Titan_Icon_SVGs.zip](https://github.com/Automattic/wp-calypso/files/6502391/Google_Titan_Icon_SVGs.zip)

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5922816).

**Pull request merged**